config/routes.rb

```diff
@@ -4,6 +4,17 @@
 
   root 'pages#show', page: 'home'
 
+  resources :bruse_files 
```

Comment:
Perhaps the removed methods from the _BruseFilesController_ can be excepted. So we don't have routes to missing methods.